orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/features/Unique.feature

```diff
@@ -61,12 +61,12 @@ Feature: As a user
 
   Scenario: The one where combination strategy is exhaustive
     Given the combination strategy is exhaustive
-    And there is a field foo
+    And there is a non nullable field foo
     And foo has type "integer"
     And foo is greater than 0
     And foo is less than 4
     And foo is anything but null
```

Comment:
all of these is anything but null's should be removed 
(sorry)

Reply:
you can probably just delete all of them, then add any back where tests have failed

Reply:
No sure I understand.  Remove all the "non nullable field" ones, what would be failing?

Reply:
as in the line 

```
    And foo is anything but null
```